Darwin/Services/StitchRemoteMongoDBService/StitchRemoteMongoDBService/Sync.swift

```diff
@@ -15,16 +15,33 @@ public class Sync<DocumentT: Codable> {
      Set the conflict resolver and and change event listener on this collection.
      - parameter conflictHandler: the conflict resolver to invoke when a conflict happens between local
      and remote events.
-     - parameter changeEventListener: the event listener to invoke when a change event happens for the
+     - parameter changeEventDelegate: the event listener to invoke when a change event happens for the
      document.
      - parameter errorListener: the error listener to invoke when an irrecoverable error occurs
      */
-    func configure<CH: ConflictHandler, CEL: ChangeEventListener>(
+    func configure(
+        conflictHandler: @escaping BlockConflictHandler<DocumentT>.ResolveConflictBlock,
```

Comment:
[style minor] I think for the sake of readability and API documentation, we should avoid using a typealias and just use the full block type. We don't use type aliases anywhere else for block arguments  It will be easier to understand, and it will let us avoid making `BlockConflictHandler` a `public` type. It can be `internal` instead.